jira/jira_tools/base.py

```diff
@@ -16,8 +17,32 @@ def __init__(self, name, description, content, args, mermaid_diagram=None):
             args=args,
             env=["JIRA_WORKSPACE_NAME"],
             secrets=["JIRA_OAUTH_TOKEN"],
-            mermaid=mermaid_diagram
+            mermaid=mermaid_diagram,
+        )
+
+
+class JiraPythonTool(Tool):
+    def __init__(
+            self, name, description, content, args, with_files=None, mermaid_diagram=None
+    ):
+        if with_files is None:
+            with_files = []
+        super().__init__(
+            name=name,
+            description=description,
+            image="python:3.12-slim",
+            on_build="""
+pip install requests > /dev/null 2>&1
```

Comment:
please remove 2>&1 at the end.
What it does is channel the stderr to stdout, that means that we wont see error either